api/v1alpha1/cephblockpool_types.go

```diff
@@ -0,0 +1,141 @@
+/*
+
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+//+kubebuilder:object:root=true
+
+// CephBlockPoolList contains a list of CephBlockPool
+type CephBlockPoolList struct {
+	metav1.TypeMeta `json:",inline"`
+	metav1.ListMeta `json:"metadata,omitempty"`
+	Items           []CephBlockPool `json:"items"`
+}
+
+//+kubebuilder:object:root=true
+//+kubebuilder:subresource:status
+
+// CephBlockPool is the Schema for the cephblockpools API
+type CephBlockPool struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	Spec   CephBlockPoolSpec   `json:"spec,omitempty"`
+	Status CephBlockPoolStatus `json:"status,omitempty"`
+}
+
+// CephBlockPoolSpec defines the desired state of CephBlockPool
+type CephBlockPoolSpec struct {
```

Comment:
Don't we need ceph cluster information to create a pool? If the policy is that there is only one ceph cluster in one namespace, then it will not be necessary.

Reply:
For now, there will be one ceph cluster per namespace, so adding ceph cluster information is not necessary.